### PR TITLE
Tweak/et 949 notice ecp rec tickets

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-949]

Linked but just formatting: https://github.com/moderntribe/tribe-common/pull/1497

7min demo: https://share.getcloudapp.com/6quxOWJb

image

[ET-949]: https://moderntribe.atlassian.net/browse/ET-949

### CODE REVIEW

File: src/Tribe/Admin/Notices.php [hunk 1]
```diff
@@ -30,9 +30,12 @@ public function maybe_display_notices() {
 			return;
 		}
 
-		$this->maybe_display_plus_commerce_notice();
+		include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
```
Comment (tr1b0t) [inline src/Tribe/Admin/Notices.php:33]: :warning: "include_once" is a statement not a function; no parentheses are required

<sub>hash: 0a933503527cba3e16625b5650ad8052</sub>

File: src/Tribe/Admin/Notices.php [hunk 2]
```diff
@@ -137,13 +212,13 @@ public function maybe_display_plus_commerce_notice() {
...
-			esc_html( 'Event Tickets Plus', 'event-tickets' )
+			esc_html__( 'Event Tickets Plus', 'event-tickets' )
 		);
 
-		$plus_commerce_providers = array(
-			esc_html( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
-			esc_html( 'Easy Digital Downloads', 'event-tickets' ) => 'easy-digital-downloads/easy-digital-downloads.php',
-		);
+		$plus_commerce_providers = [
+			esc_html__( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
```
Comment (tr1b0t) [inline src/Tribe/Admin/Notices.php:219]: :warning: Array double arrow not aligned correctly; expected 1 space(s) between ")" and double arrow, but found 12.

<sub>hash: 32ee214b91655c059bf559b1caaeea53</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 2 warning(s)

File: src/Tribe/Admin/Notices.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Admin/Notices.php:33]: remove the parentheses here.

**Review by bordoni — DISMISSED** [review]

File: readme.txt [hunk 3]
```diff
@@ -178,6 +178,7 @@ Check out our extensive [knowledgebase](https://m.tri.be/18wm) for articles on u
 
 = [5.0.4] TBD =
 
+* Tweak - Added admin notice when editing an Events Calendar Pro recurring event having tickets in classic editor. [ET-949]
```
Comment (sc0ttkclark) [inline readme.txt:181]:
```suggestion
* Tweak - Added admin notice when editing an Events Calendar Pro recurring event that has tickets in classic editor to warn about how tickets will act on recurring events. [ET-949]
```

**Review by sc0ttkclark — APPROVED** [review]

File: src/Tribe/Admin/Notices.php [hunk 4]
```diff
@@ -137,13 +211,13 @@ public function maybe_display_plus_commerce_notice() {
...
```
Comment (tr1b0t) [inline src/Tribe/Admin/Notices.php:218]: :warning: Array double arrow not aligned correctly; expected 1 space(s) between ")" and double arrow, but found 12.

<sub>hash: d95f83d09fec9bc7d9bea1da5c3b1353</sub>

**Review by tr1b0t — COMMENTED** [review]
:warning: 1 warning(s)

**Review by sc0ttkclark — APPROVED** [review]

File: src/Tribe/Admin/Notices.php [hunk 5]
```diff
@@ -137,26 +211,29 @@ public function maybe_display_plus_commerce_notice() {
...
-			esc_html( 'Event Tickets Plus', 'event-tickets' )
+			esc_html__( 'Event Tickets Plus', 'event-tickets' )
 		);
 
-		$plus_commerce_providers = array(
-			esc_html( 'WooCommerce', 'event-tickets' )            => 'woocommerce/woocommerce.php',
-			esc_html( 'Easy Digital Downloads', 'event-tickets' ) => 'easy-digital-downloads/easy-digital-downloads.php',
-		);
+		$plus_commerce_providers = [
+			'woocommerce/woocommerce.php'                       => __( 'WooCommerce', 'event-tickets' ),
```
Comment (tr1b0t) [inline src/Tribe/Admin/Notices.php:218]: :warning: Array double arrow not aligned correctly; expected 1 space(s) between "'woocommerce/woocommerce.php'" and double arrow, but found 23.

<sub>hash: d2b1653e479fc81c3157ee68c7012cb6</sub>

File: src/Tribe/Admin/Notices.php [hunk 6]
```diff
@@ -137,26 +211,29 @@ public function maybe_display_plus_commerce_notice() {
...
 
-		foreach ( $plus_commerce_providers as $provider => $path ) {
+		foreach ( $plus_commerce_providers as $path => $provider ) {
 			if ( ! is_plugin_active( $path ) ) {
 				continue;
 			}
 
 			$message = sprintf(
-				__( 'Event Tickets does not support ticket sales via third party ecommerce plugins. If you want to sell tickets with %1$s, please purchase a license for %2$s.' ),
+				__( 'Event Tickets does not support ticket sales via third party ecommerce plugins. If you want to sell tickets with %1$s, please purchase a license for %2$s.', 'event-tickets' ),
```
Comment (tr1b0t) [inline src/Tribe/Admin/Notices.php:228]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: cd86fb304e58c83fce6b46a0457cb5cf</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 1 error(s)
:warning: 1 warning(s)

**Comment by sc0ttkclark** [comment]
Holding merge for after https://github.com/moderntribe/event-tickets/pull/1878 is merged

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**